### PR TITLE
Add scams from ChainPatrol

### PR DESCRIPTION
**Pull request by NikitaVr**
## Scam links
- [ark-defi.pro](https://ark-defi.pro)

## Screenshots

---

## Scam links
- [starknet.deals](https://starknet.deals)

## Screenshots

---

## Scam links
- [arblrtrum-airdrop.online](https://arblrtrum-airdrop.online)

## Screenshots

---

## Scam links
- [survey-arbitrum.info](https://survey-arbitrum.info)

## Screenshots

---

## Scam links
  - [arbdrop.network](https://arbdrop.network)

---

## Scam links
  - [twitter.com/cormier50424](https://twitter.com/cormier50424)

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -513,6 +513,7 @@
     "launchpad.ethereum.org"
   ],
   "blacklist": [
+    "twitter.com/cormier50424",
```
Comment (umar-ahmed) [inline src/config.json:516]: This is a bug. Testing a new workflow. Will revert this change

**Review by deshvin — APPROVED** [review]
